src/pause-play/js/main.js

```diff
@@ -124,11 +106,21 @@ class TestRunner {
     this.lastIterationTime;
   }
 
-  addPeerConnection() {
-    const videoElement = this.feedTable.addNewCell();
-    this.videoElements.push(videoElement);
+  addPeerConnection(elementType) {
+    const element = this.feedTable.addNewCell(elementType);
+    const constraints = {audio: true};
+    if (elementType === 'video') {
+      constraints.video = {
+        width: {exact: 300},
+      };
+    } else if (elementType === 'audio') {
+      constraints.video = false;
+    } else {
+      console.error("elementType must be one of 'audio' or 'video'");
```

Comment:
You also need to exit here, otherwise it will continue with an unsupported elementType. Throw an error instead of the console.error statement with an error message. E.g `throw new Error("elementType must be one of 'audio' or 'video'")`